README.md

```diff
@@ -114,11 +114,13 @@ A few rules to clear up this ambiguity:
 - Relative paths referenced on the command line are relative to the current working directory the user is in
 
 For additional context, take a look at [paths examples](PATHS.md)
-## Tags Overview
-Tags can be used to only use a subset of releases when running helmfile. This is useful for large helmfiles with releases that are logically grouped together.
+## Labels Overview
+Labels can be used to only use a subset of releases when running helmfile. This is useful for large helmfiles with releases that are logically grouped together.
 
-Tags are simple key value pairs that are an optional field of the release spec. When filtering by tag, the search can be inverted. `tier!=backend` would match all releases that do NOT have the `tier: backend` tag. `tier=fronted` would only match releases with the `tier: frontend` tag.
+Labels are simple key value pairs that are an optional field of the release spec. When filtering by label, the search can be inverted. `tier!=backend` would match all releases that do NOT have the `tier: backend` label. `tier=fronted` would only match releases with the `tier: frontend` label.
 
-Multiple tags can be specified using `,` as a separator. A release must match all tags in order to be selected for the final helm command. 
+Multiple labels can be specified using `,` as a separator. A release must match all labels in order to be selected for the final helm command. 
 
-The `tags` parameter can be specified multiple times. Each parameter is resolved independently so a release that matches any parameter will be used. 
\ No newline at end of file
+The `labels` parameter can be specified multiple times. Each parameter is resolved independently so a release that matches any parameter will be used. 
+
+`--selector tier=frontend --selector tier=backend` will select all the charts
```

Comment:
this should be --labels right?

Reply:
`--selector`,`-l` is consistent with `kubectl` and how it selects objects using labels. I'd used `-l` myself previously but I think having a common operation across tools is desirable.